Allow use of esbuild 0.15.3 (<=0.15)

Currently, if you try and upgrade to esbuild to version 0.15.x, npm complains that serverless-esbuild can only use esbuild >=0.8 <0.15:
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: @vidigami/serverless@1.0.1
npm ERR! Found: esbuild@0.15.3
npm ERR! node_modules/esbuild
npm ERR!   dev esbuild@"0.15.3" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer esbuild@">=0.8 <0.15" from serverless-esbuild@1.32.6
npm ERR! node_modules/serverless-esbuild
npm ERR!   dev serverless-esbuild@"^1.32.6" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/robblovell/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/robblovell/.npm/_logs/2022-08-15T21_22_54_474Z-debug-0.log
```
**Describe the solution you'd like**
I would like to be able to use esbuild version 0.15.x

**Describe alternatives you've considered**
Otherwise, I am forced to use esbuild version 0.14.54.

--- a/package.json
+++ b/package.json
@@ -82,7 +82,7 @@
     "semver": "^7.3.5"
   },
   "peerDependencies": {
-    "esbuild": ">=0.8 <0.15"
+    "esbuild": ">=0.8 <0.16"
   },
   "lint-staged": {
     "*.ts": [

